src/Compilers/CSharp/Portable/Syntax/InternalSyntax/SyntaxToken.cs

```diff
@@ -84,6 +84,11 @@ public override bool IsToken
             }
         }
 
+        internal bool IsVar
```

Comment:
The one I found was on `TypeSyntax`. But in my case I need to check for `var` before it was recognized as a type.